scripts/github_api

```diff
@@ -0,0 +1,114 @@
+#!/bin/python
+from functools import partial
+import argparse
+import json
+import re
+import requests
+import sys
+
+CLIENT_ERROR = 422
+API_BASE_URL = "https://api.github.com/"
+
+
+# see _setup_auth function below
+request = None
```

Comment:
We could define this function by raising an exception that informs `setup_auth` must be called first.